content/id/docs/concepts/policy/resource-quotas.md

```diff
@@ -0,0 +1,622 @@
+---
+title: Resource Quota
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Saat beberapa pengguna atau tim berbagi sebuah klaster dengan jumlah Node yang tetap,
+ada satu hal yang perlu diperhatikan yaitu suatu tim dapat menggunakan sumber daya
+lebih dari jatah yang mereka perlukan.
+
+_Resource Quota_ (kuota sumber daya) adalah sebuah alat yang dapat digunakan oleh
+administrator untuk mengatasi hal ini.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+Sebuah Resource Quota, didefinisikan oleh objek API `ResourceQuota`, menyediakan batasan-batasan
+yang membatasi konsumsi gabungan sumber daya komputasi untuk tiap Namespace. Resource Quota dapat
+membatasi jumlah objek yang dapat dibuat dalam sebuah Namespace berdasarkan tipenya, maupun jumlah
+seluruh sumber daya komputasi yang dapat dipakai oleh sumber daya API (misalnya Pod) di Namespace
+tersebut.
+
+Resource Quota bekerja sebagai berikut:
+
+- Tim-tim berbeda bekerja pada Namespace yang berbeda pula. Saat ini hal ini belum diwajibkan,
```

Comment:
```suggestion
- Tim-tim berbeda bekerja pada Namespace yang berbeda pula. Sekarang hal ini belum diwajibkan,
```
how about using _sekarang_, so there will be one _ini_ word?

Reply:
Agreed. Done!

Reply:
@wahyuoi